plugins/git-commit/git-commit.plugin.zsh

```diff
@@ -14,6 +14,56 @@ _git_commit_aliases=(
   'wip'
 )
 
+_git_commit_flags=(
```

Comment:
Could you `typeset -a _git_commit_flags` before? It shouldn't be necessary, but it's always nice to do it!

Reply:
Yes I can for sure!